src/World.cpp

```diff
@@ -544,15 +544,18 @@ void cWorld::Start(void)
 	if (GetDimension() == dimOverworld)
 	{
 		AString MyNetherName = GetName() + "_nether";
-		AString MyEndName = GetName() + "_end";
+		AString MyEndName = GetName() + "_the_end";
 		if (cRoot::Get()->GetWorld(MyNetherName) == nullptr)
 		{
 			MyNetherName = "";
 		}
 		if (cRoot::Get()->GetWorld(MyEndName) == nullptr)
-		{
-			MyEndName = "";
-		}
+ 		{
+ 			MyEndName = GetName() + "_end";
+ 			if (cRoot::Get()->GetWorld(MyEndName) == nullptr)
+ 			MyEndName = "";
+ 		}
+
```

Comment:
There's indentation by spaces (yuck) here, and missing braces and indent around the `if`-controlled block